instagram/src/components/SignInPage/SignInPage.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "./SignInPage.css";
+
+class SignInPage extends React.Component {
+  state = {
+    user: "",
+    password: ""
+  };
+
+  handleChange = e => {
+    console.log(e.target.value);
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
+  formSubmitHandler = e => {
+    e.preventDefault();
+    const user = this.state.username;

```

Comment:
You're on the right track with saving the username in localStorage. To go even further, we can use this localStorage item to render the user's name when adding a comment - currently hardcoded to "marcelo."

I'm sure this is unintentional (happens to me too) but first we need to fix this typo
```suggestion
    const user = this.state.user;
```